engine/admin/index.md

```diff
@@ -75,6 +75,133 @@ restart Docker. This method works for every Docker platform. The following
 }
 ```
 
+## Isolate containers within a user namespace
+
+Linux namespaces provide isolation for running processes, limiting their access
+to system resources without the running process being aware of the limitations.
+For more information on Linux namespaces, see
+[Linux namespaces](https://www.linux.com/news/understanding-and-securing-linux-namespaces){: target="_blank" class="_" };
+
+The best practice for preventing privilege-escalation attacks from within a
+container is to configure your container's applications to run as unprivileged
+users. For containers whose processes must run as the `root` user within the
+container, you can re-map this user to a less-privileged user on the Docker
+host.
+
+This re-mapping is relatively transparent to the container, but introduces some
+configuration complexity when the container needs access to privileged resources
```

Comment:
we can remove "privileged" here; also files owned by "regular" users will be inaccessible.

The goal of user-namespaces is to give the user inside the container no access to _anything_ on the host (i.e., if the process _would_ be able to escape the container, it's basically harmless, as it won't be able to do anything).